### ISSUE TITLE
Wrong filename shown for authentication webhook config in provisioning logging

### ISSUE THREAD

**Issue by superseb** [rancher/rancher#19444]
**What kind of request is this (question/bug/enhancement/feature request):**
bug

**Steps to reproduce (least amount of steps as possible):**
- Create custom cluster with default authorized cluster endpoint enabled
- See logging for provisioning, shows `/etc/kubernetes/cloud-config`

**Result:**
```
2019/04/08 15:02:52 [INFO] cluster [c-2nrn8] provisioning: [file-deploy] Deploying file '/etc/kubernetes/kube-api-authn-webhook.yaml' to node [167.99.89.93]
2019/04/08 15:02:52 [INFO] cluster [c-2nrn8] provisioning: [file-deploy] Successfully started [file-deployer] container on host [167.99.89.93]
2019/04/08 15:02:53 [INFO] cluster [c-2nrn8] provisioning: [remove/file-deployer] Successfully removed container on host [167.99.89.93]
2019/04/08 15:02:53 [INFO] cluster [c-2nrn8] provisioning: [/etc/kubernetes/cloud-config] Successfully deployed authentication webhook config Cluster nodes
```

**Comment by jiaqiluo** [comment]
Wait for rancher/rancher to upgrade the vendor list to include the fix.

**Comment by sangeethah** [comment]
Removed "status/to-test" since we are waiting for Rke fixes to be in rancher builds.

**Comment by alena1108** [comment]
Can be tested with latest master image and `rancher/rancher:v2.2.3-rc2`

**Comment by jiaqiluo** [comment]
The bug fix is validated in Rancher master e91e9dcce7d78b9b5d09cfb8cf71655de2f623bd

But the fix (https://github.com/rancher/rancher/commit/99d211a9a1199b57427a945fb7fefa1c867c6f43) is not cut into v2.2.2-rc2 

Wait for another

**Comment by jira-sync-svc** [comment]
➤ Jack Luo commented:

The bug fix is validated in Rancher v2.2.3-rc3
